08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,98 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from 'react';
+import InputText from '../input_text';
+
+/**
+ * inputText1 time=1 value=5 3600 1
+ * inputText2 time=60 value=5 60
+ * inputText3 time=3600 1
+ */
+
+export default class TimeConverter extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {

```

Comment:
Do you really need 3 state variables? after all both minutes and hours can be described with seconds